src/components/Label/index.js

```diff
@@ -10,6 +10,7 @@ const Label = glamorous.label(
     display: "flex",
     flexDirection: "column",
     boxSizing: "border-box",
+    width: "100%",
```

Comment:
can this change be made independent to the flex change? Or is it specifically tied to the css for the grid?